lib/logstash/outputs/google_bigquery.rb

```diff
@@ -181,6 +182,7 @@ def register
     @schema = LogStash::Outputs::BigQuery::Schema.parse_csv_or_json @csv_schema, @json_schema
     @bq_client = LogStash::Outputs::BigQuery::StreamingClient.new @json_key_file, @project_id, @logger
     @batcher = LogStash::Outputs::BigQuery::Batcher.new @batch_size, @batch_size_bytes
+    @stopping = Concurrent::AtomicBoolean.new
```

Comment:
nitpick: I suggest to pass `false`, even though it is the default, to make it explicit that we start in the `false` state - it avoid someone looking at the code wonders if the default it `true` or `false`.
```suggestion
    @stopping = Concurrent::AtomicBoolean.new(false)
```